canonical-websites/www.ubuntu.com
# Add image widths for performance

## Done

- for most pages, excluding /engage and /takeovers I added heights and/or widths to most images with both the code (eg `width="144`)and with the asset server parameter (eg `?w=144`)
- removed some includes that are no longer used
- fixed some links to https
- added `class=p-link--external` to a few missing links

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Click through the pages updated to see if they still look ok

## Issue / Card

Fixes #4601
